3rd-party-tools/plink-regenie/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:latest 
```

Comment:
Can we change this to python:3.10?

Reply:
Discussed this with Liz offline and we agreed to keep Ubuntu. Updated to select a version.